Src/Sentry.Xamarin/Internals/Session/DeviceActiveLogger.uwp.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Sentry.Internals.Session
+{
+    internal class DeviceActiveLogger : IDeviceActiveLogger
```

Comment:
Do you need this class to be per platform? uwp and ios at least are the same

Reply:
iOS and UWP work the same way, but, on Android, it works slightly different, that's why I opted for that.